cmd/metal-api/internal/service/switch-service.go

```diff
@@ -132,6 +132,15 @@ func (r *switchResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", v1.SwitchNotifyResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.POST("/migrate").
+		To(editor(r.migrate)).
```

Comment:
I think we should raise this to admin, which is typically an operator. Components like metal-core are editor.

```suggestion
		To(admin(r.migrate)).
```